exchanges/orderbook/orderbook_test.go

```diff
@@ -507,108 +455,104 @@ func TestSorting(t *testing.T) {
 }
 
 func deploySliceOrdered() Levels {
```

Comment:
🐒  There's nothing *but* a sorted fixture for levels, and "deploy" doesn't carry meaning. Nor "slice" really.
```suggestion
func levelsFixture() Levels {
```

Reply:
This statement from me was not true. Not sure how I missed the Unordered fixture.
But I still think this works, and it's just the Unordered/Random one we need to rename

Reply:
https://github.com/thrasher-corp/gocryptotrader/pull/1914/commits/144a6c9068203a83b4d2d08079768bfd90cda3c4